test/api/v3/integration/groups/POST-groups_groupId_join.test.js

```diff
@@ -90,7 +115,7 @@ describe('POST /group/:groupId/join', () => {
     });
   });
 
-  context('Accepting invitation to a party', () => {
+  context('Joining a party', () => {
     let user, invitedUser, party;
```

Comment:
I disabled the requirement for multiple lines in tests. (see https://github.com/HabitRPG/habitrpg/blob/023ee44ddcae865b5a52fc2f4f858b4346c82ae9/test/.eslintrc#L3)

It's not an es6 thing, it's just a style thing. Since tests regularly need a bunch of undefined variables, I thought it was appropriate to remove the rule for it. If y'all have strong feelings about it, I can re-enable and fix the tests.